src/js/tls.js

```diff
@@ -3,85 +3,110 @@
 require('crypto');
 
 var net = require('net');
+var tcp = require('tcp');
 var util = require('util');
-var EventEmitter = require('events').EventEmitter;
+/** @type tls.TlsWrap */
```

Comment:
What's the purpose of this line for?

Reply:
This annotation indicates what type of variable declared next line is